server/tweetSearch.js

```diff
@@ -110,34 +113,41 @@ module.exports = function(client) {
         return tweetStore;
```

Comment:
That's actually a different function, GitHub just folds the unedited lines between the end of addTweetItem and getTweetStore out